### ISSUE TITLE
File Permissions and Ownership page

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#489]
This page has not been published yet, but I think it's important to help people with updating core through the UI (which is probably why it was created). https://github.com/backdrop/backdrop-issues/issues/3271

https://backdropcms.org/file-permissions-and-ownership

I could not get these commands to work. I don't think you want the "chmod" at the beginning. 

_# Set the permissions for files and directories.
chmod find . -type f -exec chmod 664 '{}' \;
chmod find . -type d -exec chmod 755 '{}' \;_

Rather, I think you want this:

_# Set the permissions for files and directories.
find . -type f -exec chmod 664 '{}' \;
find . -type d -exec chmod 755 '{}' \;_

This happens twice on the page.

**Comment by quicksketch** [comment]
Thanks @stpaultim! I fixed that typo and published the page. @serundeputy and I had some back and forth on whether we should make further changes, but let's publish what we have and adjust if needed as we go.

I published that page, and it's available now. Thanks!

**Comment by stpaultim** [comment]
The changes I suggested were made to the more strict version of the settings, but not to the looser version. I went ahead and made these changes. 

I also changed this. It was not working for me as it was written. I believe that the -R must come before the kris:kris for this to work (in my tests it was true). 

`# Set the ownership of the current directory and all children.
chown kris:kris -R .`

To

`# Set the owner and group for all files.
chown -R kris:kris .`
